.travis.yml

```diff
@@ -47,7 +47,7 @@ matrix:
 
 before_install:
   - echo cakephp version && tail -1 VERSION.txt
-  - if [[ ${TRAVIS_PHP_VERSION} != "7.3" ]]; then phpenv config-rm xdebug.ini; fi
+  - if [[ $CODECOVERAGE != 1 ]]; then phpenv config-rm xdebug.ini; fi
```

Comment:
The PHP 7.3 was done because 7.3 compatible xdebug didn't exist, does it now? Also for code codecoverage we use `phpdbg` and don't need xdebug for that job either.

So the line should be just changed to `phpenv config-rm xdebug.ini` is PHP 7.3 check isn't required.

Reply:
My findings say it works this way there too.

Reply:
Where's "there"? :)

Reply:
Travis green shows no check necessary. That is better than doing check and missing to remove it.

Reply:
I said to remove remove check and just use `phpenv config-rm xdebug.ini` :slightly_smiling_face:.

Reply:
We can sure try :)

Reply:
But wait.. coverage runs on 7.1 and that needs it. So no, I think the current way is correct and mergable.

Reply:
Repeating what I said above:

> Also for code codecoverage we use phpdbg and don't need xdebug for that job either.

Reply:
;)